pkgs/applications/virtualization/rust-hypervisor-firmware/default.nix

```diff
@@ -0,0 +1,57 @@
+{ lib
+, fetchFromGitHub
+, makeRustPlatform
+, hostPlatform
+, targetPlatform
+, lld
+}:
+
+let
+  arch = targetPlatform.qemuArch;
+
+  target = ./. + "/${arch}-unknown-none.json";
```

Comment:
Breaks nixpkgs-review on aarch64-darwin:

```
error: opening file '/Users/user/.cache/nixpkgs-review/pr-222011-2/nixpkgs/pkgs/applications/virtualization/rust-hypervisor-firmware/aarch64-unknown-none.json': No such file or directory
```

Reply:
Should `lib.platforms` rather be the host platform, `x86_64-linux`?